chef/cookbooks/ceph/recipes/osd.rb

```diff
@@ -68,7 +68,8 @@
 
   if is_crowbar?
     node.set["ceph"]["osd_devices"] = [] if node["ceph"]["osd_devices"].nil?
-    unclaimed_disks = BarclampLibrary::Barclamp::Inventory::Disk.unclaimed(node).sort
+    min_size_blocks = node["ceph"]["osd"]["min_size_gb"] * 1024 * 1024 * 2
+    unclaimed_disks = BarclampLibrary::Barclamp::Inventory::Disk.unclaimed(node).sort.select {|d| d.size >= min_size_blocks}
```

Comment:
> Yeah, seems to work fine (i.e. it returns the disk size in 512 byte blocks)

I guess I'm confused, but if we're comparing size in blocks, then how do we know that this is 20GB?

FWIW, we used to have OHAI-485-block_device-logical-block-size.patch in ohai to provide the size (not in blocks).